src/ekat/ekat_physical_constants.hpp

```diff
@@ -0,0 +1,105 @@
+#ifndef EKAT_PHYSICAL_CONSTANTS
+#define EKAT_PHYSICAL_CONSTANTS
+
+#include "ekat/util/ekat_units.hpp"
+
+namespace ekat {
+
+template <typename RealType=double>
+class PhysicalConstant {
+  public:
+    enum Op {Times, Divide};
+
+    // no default
+    PhysicalConstant() = delete;
+
+    PhysicalConstant(const PhysicalConstant<RealType>&) = default;
+
+    // Constructor for single value (i.e., NIST std)
+    PhysicalConstant(const RealType& nvalue, const ekat::units::Units& units, const std::string& name) :
+      m_nist_val(nvalue), m_e3sm_val(nvalue), m_units(units), m_name(name), m_info("") {}
+
+    // Constructor for single value (i.e., NIST std)
+    PhysicalConstant(const RealType& nvalue, const ekat::units::Units& units, const std::string& name,
+      const std::string& info) :
+      m_nist_val(nvalue), m_e3sm_val(nvalue), m_units(units), m_name(name), m_info(info) {}
+
+
+    // Constructor for two different values (i.e., when E3SM value and NIST value are different)
+    PhysicalConstant(const RealType& evalue, const RealType& nvalue, const ekat::units::Units& units,
```

Comment:
Note (for here and the rest of the file): you don't need to prepend `ekat::`, since you are already in the ekat namespace. That might help keeping lines shorter, which is helpful for the reader.